app/models/ability.rb

```diff
@@ -95,8 +102,9 @@ def initialize(user)
       # inwhich case we need to confirm membership and permission
 
       group.is_parent? ||
-       ((user_is_member_of?(group.parent.id) && group.parent.members_can_create_subgroups?)) ||
-       user_is_admin_of?(group.parent.id)
+      user_is_admin_of?(group.parent.id) ||
```

Comment:
(feel free to ignore if annoying :) )

We should be doing `parent_id` instead of `parent.id`, which'll save us a DB request